files/dr.conf

```diff
@@ -0,0 +1,10 @@
+[generate_vars]
+site=http://localhost:8080/ovirt-engine/api
+username=admin@internal
+password=12
+ca_file=/etc/pki/ovirt-engine/ca.pem
+output_file=/var/lib/ovirt-ansible-disaster-recovery/mapping_vars.yml
```

Comment:
does this directory exists?

Reply:
not currently but I would like to generate it once the user install the RPM.
I think that it will be more appropriate place, unless you have other suggestion.

Reply:
I changed it to be relative to what there is in the examples for now, to keep it simple